lib/catindexinfo.go

```diff
@@ -8,50 +8,53 @@ import (
 
 var ErrInvalidIndexLine = errors.New("Cannot parse indexline")
 
-// Create an IndexInfo from the string _cat/indices would produce
+//Create an IndexInfo from the string _cat/indices would produce
+//EX: health status index    pri rep docs.count docs.deleted store.size  pri.store.size
+//green open logs-2015-06-19   2 0   135389346  0            53048922233 53048922233
 func NewCatIndexInfo(indexLine string) (catIndex *CatIndexInfo, err error) {
 	split := strings.Fields(indexLine)
-	if len(split) < 4 {
+	if len(split) < 5 {
```

Comment:
this breaks backwards compatibility with 1.3, but I guess that is the price we pay.  The only other way would be to check the contents of split[1] and see if it matched the pattern "open" "closed" then we assume this behavior, if not we use the old behavior.